cf/field.py

```diff
@@ -15823,6 +15823,7 @@ def regrids(
                     f"Can't regrid {self!r} with regridding operator "
```

Comment:
I  think we should have a `srcgrid.destroy()` just before the raising of the `ValueError`, too. This would be 2 lines above here, but the GitHub editor didn't seem to want to show me that line!)

Reply:
Thank you David, I had not thought of this one!